README.md

```diff
@@ -119,28 +119,71 @@ multiple labels can refer to the same bundle via its commit hash.
 
 Current use of Datamon at One Concern with respect to intra-Argo workflow usage relies on the
```

Comment:
Just a question here: I do understand that our use case is with ARGO, but I believe datamon is quite independent from ARGO, isn't it? (no change requested for that)

Reply:
(direct-message channel)

there's a small is-ought gap here:  datamon should be independent of argo, but practically, most of our users are folks writing argo pipelines, and we want to provide conveniences.
https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain
is meaningful here.
if datamon ever gets to be a popular thing, i'd like to have a [clearly delineated] porcelain layer, possibly in scheme or s'th.